neuromation/cli/main.py

```diff
@@ -196,6 +197,17 @@ def print_options(
         "together with option '--trace'. On by default."
     ),
 )
+@option(
+    "--skip-stats/--no-skip-token",
```

Comment:
Looks like a typo: should be `--no-skip-stats`

Reply:
You are right, thanks!